### ISSUE TITLE
Accessibility - on sign up, the "terms of use" checkbox is not in the correct tab order

### ISSUE THREAD

**Issue by kaitlinnewson** [IQSS/dataverse#5355]
Tested in demo dataverse, v. 4.9.4., using both Firefox and Chrome.

When a user signs up for the first time, the checkbox beside 'I have read and accept the Dataverse General Terms of Use as outlined above.' is not in the correct tab order, which should be after the 'Position' field.

**Comment by pameyer** [comment]
Oddly enough, tab ordering was correct in lynx.  Appeared to be due to specifying `tabindex` in `dataverseuser.xhtml`, but not the included `termsofuse.xhtml` (with no `tabindex`).  Allowing document structure to dictate tab order appears to resolve the issue; will defer to more UI minded folks if this has adverse consequences.

**Comment by mheppler** [comment]
Thanks for the pull request, @pameyer. I will take a look!

**Comment by mheppler** [comment]
Removing the `tabindex` attribute appears to have resolved the tabbing issue on the Sign Up/Create Account pg, as well as the Password Reset pg and Convert Account/Shib pg, where the `termsofuse.xhtml` fragment is also included.

NOTE: If testing in Safari, and you can not tab to the checkbox or "Save Changes" and "Cancel" form buttons, it is because you need to change your browser preferences. Go to **Preferences > Advanced** and under **Accessibility** check the _"Press Tab to highlight..."_ option.

Sending this over to QA. Thanks @pameyer and @kaitlinnewson for helping improve this.
